src/smt/preprocess_proof_generator.cpp

```diff
@@ -56,67 +75,99 @@ std::shared_ptr<ProofNode> PreprocessProofGenerator::getProofFor(Node f)
   // make CDProof to construct the proof below
   CDProof cdp(d_pnm);
 
+  Trace("smt-pppg") << "PreprocessProofGenerator::getProofFor: input " << f
+                    << std::endl;
   Node curr = f;
   std::vector<Node> transChildren;
+  std::unordered_set<Node, NodeHashFunction> processed;
```

Comment:
It'd be good to say, before the end of this method, what this loop is supposed to be doing